### DISCUSSION TITLE
Explore Cloud Custodian's Cache with VisiData

### DISCUSSION THREAD

**Discussion by ajkerrigan** [cloud-custodian/cloud-custodian#8334] [Tips and Tricks]
Since #7659, Custodian's default cache has been a [SQLite](https://sqlite.org/) database stored at `~/.cache/cloud-custodian.cache`, which expires data after 15 minutes.

Inside that SQLite database are two columns: `key` and `value`, each of which are [pickled](https://docs.python.org/3/library/pickle.html) blobs.

If you've ever wanted to poke at cache data after a policy run, [here's](https://gist.github.com/ajkerrigan/1214d4632b9e46cfa77c024d7c12f23d) one way to do that with [VisiData](https://www.visidata.org/).

**Comment by klauern** [comment]
What version of cloud custodian has the cache?  I'm not finding it on my instance, but it could be that we're just too far behind.

**Reply by kapilt** [reply]
I think its been around since late august 2022, so about a half dozen releases.
